experiment/slack-oncall-updater/main.go

```diff
@@ -26,6 +26,31 @@ import (
 	"strings"
 )
 
+// githubToSlack maps (lowercase!) GitHub usernames to Slack user IDs.
+// Add yourself when you join the oncall rotation.
+// You can mostly easily find your user ID by visiting your Slack profile, clicking "...",
+// and clicking "Copy user ID".
+var githubToSlack = map[string]string{
+	"amwat":          "U9B1P2UGP",
+	"chases2":        "UJ9R0FWD6",
+	"cjwagner":       "U4QFZFMCM",
+	"bentheelder":    "U1P7T516X",
```

Comment:
Should we alphabetize on the off chance that this gets that large?

Reply:
done.